assets/javascript/apps/pipeline/nodes/GetInputWidget.tsx

```diff
@@ -86,150 +81,7 @@ export const getInputWidget = (params: InputWidgetParams) => {
       schema={params.schema}
       nodeParams={params.params}
       required={params.required}
+      getFieldError={getFieldError}
```

Comment:
Why not use the existing `inputError` prop?

Reply:
Because the history widget represents 2 fields so it needs access to the error for both fields.